### ISSUE TITLE
[ue4-docker build] Error: failed to build image "adamrehn/ue4-source:4.24.3-opengl-ubuntu18.04".

### ISSUE THREAD

**Issue by aharoJ** [carla-simulator/carla#6646]
I was following the [Building the Docker images](https://github.com/carla-simulator/carla/tree/master/Util/Docker#building-the-docker-images) but I get the error `  [ue4-docker build] Error: failed to build image "adamrehn/ue4-source:4.24.3-opengl-ubuntu18.04"  ` 

Can anyone point me to the right direction? 

I modified the script to `ue4-docker build 4.24.3 --no-minimal` from the original `ue4-docker build 4.24.3 --no-engine --no-minimal` because that argument no longer exist.
-> shell error with old script: e4-docker build: error: unrecognized arguments: --no-engine

New Script:
`ue4-docker build 4.24.3 --no-minimal`
ERROR: [ue4-docker build] Error: failed to build image "adamrehn/ue4-source:4.24.3-opengl-ubuntu18.04".

my `ue4-docker info`:
ue4-docker version:         0.0.108 (latest available version is 0.0.108)
Operating system:           Linux (Ubuntu 20.04.6 LTS, 5.15.0-76-generic)
Docker daemon version:      24.0.4
NVIDIA Docker supported:    Yes
Maximum image size:         No limit detected
Available disk space:       168.47 GiB
Total system memory:        62.64 GiB physical, 2 GiB virtual
CPU:                        8 physical, 16 logical (x86_64)

**Comment by 2019211753** [comment]
Did u solve it? I also met the question. Thanks.

**Comment by aharoJ** [comment]
@2019211753 no, I ended up downloaded carla from source binary and it fixed it.

**Comment by 2019211753** [comment]
> @2019211753 no, I ended up downloading carla from source binary and it fixed it.

yes, i also use the carla from source afterwards 😄

**Comment by Dragoadri** [comment]
How do you install carla from source binary then ??and where??

**Comment by YangyangFu** [comment]
I think the issue is related to this: https://forums.unrealengine.com/t/upcoming-disruption-of-service-impacting-unreal-engine-users-on-github/1155880/33.

**Comment by xiaox2600** [comment]
I changed the ue4-docker version to 0.0.100, and it works. I think the new version of ue4-docker has removed the option --no-engine
